setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import os
+
+from setuptools import setup, find_packages
+
+setup(
+    name='clcache',
+    description='MSVC Complier cache',
+    author='frerich',
+    author_email='frerich@users.noreply.github.com',
```

Comment:
Please use the same email address given in the README file, i.e. raabe@froglogic.com